Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstandingCustomMesh.cs

```diff
@@ -118,6 +123,8 @@ public void Commit()
                     MeshObject.SetTriangles(tris, 0);
                     MeshObject.RecalculateNormals();
                     MeshObject.RecalculateBounds();
+                    SpatialCollider.sharedMesh = null;
+                    SpatialCollider.sharedMesh = MeshObject;
```

Comment:
Just one last thing. Do we need to set the `sharedMesh` to null before we set it?